nexus/db-model/src/u32.rs

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We already have a `SqlU32` type in `nexus/db-model/src/unsigned.rs`. Is this different?

Reply:
Removed.